src/main/java/com/revature/dao/ReimbursementDao.java

```diff
@@ -0,0 +1,32 @@
+package com.revature.dao;
+
+import java.util.List;
+
+import com.revature.models.reimbursement.Reimbursement;
+
+public class ReimbursementDao implements IDao<Reimbursement> {
+
+    @Override
+    public List<Reimbursement> selectAll() {
+      // TODO Auto-generated method stub
+      return null;
+    }
+
```

Comment:
Needs an `insert` method since it is already specified in the `IDao` interface

Reply:
Competed with a Genatic implimentation